test/testsets/integrations/RCall_refresh.jl

```diff
@@ -0,0 +1,36 @@
+include("../../utilities/prelude.jl")
+
+@testset "RCall refresh" begin
+    s = Server()
+
+    base_file =
+        joinpath(@__DIR__, "..", "..", "examples", "integrations", "RCall_refresh.qmd")
+    copy = joinpath(dirname(base_file), "RCall_refresh_copy.qmd")
+
+    function _cells(file)
+        io = IOBuffer()
+        run!(s, file; output = io)
```

Comment:
If you don't pass an `output` it'll just return the JSON parsed object, should allow you to avoid this buffer usage.